tldr.py

```diff
@@ -267,6 +268,8 @@ def get_page(
             except URLError:
                 if not PAGES_SOURCE_LOCATION.startswith('file://'):
                     raise
+            except TimeoutError:
+                continue
```

Comment:
I'm not sure we should be skipping this, in that we don't skip if we get a 500 `HTTPError`. Something about the upstream is broken in a way that we cannot connect to it, and I don't think we should continue on and potentially serve the wrong page.

Reply:
I've read some more info about that and I've noticed that in fact a `URLError` (and not a `TimeoutError`) is raised when the timeout expires. Consequently I've deleted that part of the code and committed it. The behavior is that whenever a request is not answered an error message is shown to the user. Sorry for the confusion.